dev/tests/integration/testsuite/Magento/Framework/Api/ExtensionAttribute/JoinProcessorTest.php

```diff
@@ -243,12 +244,13 @@ public function testProcessSqlSelectVerification()
         $reviews = $this->appResource->getTableName('reviews');
         $expectedSql = <<<EXPECTED_SQL
 SELECT `e`.*,
-     `extension_attribute_stock_item`.`qty` AS `extension_attribute_stock_item_qty`,
-     `extension_attribute_reviews`.`comment` AS `extension_attribute_reviews_comment`,
-     `extension_attribute_reviews`.`rating` AS `extension_attribute_reviews_rating`,
-     `extension_attribute_reviews`.`date` AS `extension_attribute_reviews_date` FROM `$catalogProductEntity` AS `e`
- LEFT JOIN `$catalogInventoryStockItem` AS `extension_attribute_stock_item` ON e.id = extension_attribute_stock_item.id
- LEFT JOIN `$reviews` AS `extension_attribute_reviews` ON e.id = extension_attribute_reviews.product_id
+     `extension_attribute_table_cataloginventory_stock_item_on_id_reference_id`.`qty` AS `extension_attribute_stock_item_qty`,
+     `extension_attribute_table_reviews_on_id_reference_product_id`.`comment` AS `extension_attribute_reviews_comment`,
+     `extension_attribute_table_reviews_on_id_reference_product_id`.`rating` AS `extension_attribute_reviews_rating`,
+     `extension_attribute_table_reviews_on_id_reference_product_id`.`date` AS `extension_attribute_reviews_date`,
+     `extension_attribute_table_reviews_on_id_reference_product_id`.`id` AS `extension_attribute_review_id_id` FROM `$catalogProductEntity` AS `e`
+ LEFT JOIN `$catalogInventoryStockItem` AS `extension_attribute_table_cataloginventory_stock_item_on_id_reference_id` ON e.id = extension_attribute_table_cataloginventory_stock_item_on_id_reference_id.id
+ LEFT JOIN `$reviews` AS `extension_attribute_table_reviews_on_id_reference_product_id` ON e.id = extension_attribute_table_reviews_on_id_reference_product_id.product_id
```

Comment:
Please create a separate test asserting that two extension attributes from one table are obtained with a single left join.